frontend/src/pages/projects/screens/projects/EmptyProjects.tsx

```diff
@@ -21,36 +23,58 @@ type EmptyProjectsProps = {
 
 const EmptyProjects: React.FC<EmptyProjectsProps> = ({ allowCreate }) => {
   const navigate = useNavigate();
-  const isJupyterEnabled = useCheckJupyterEnabled();
+
   return (
-    <EmptyState>
+    <EmptyState variant="lg">
       <EmptyStateHeader
         data-testid="no-data-science-project"
-        titleText="No data science projects yet."
-        icon={<EmptyStateIcon icon={CubesIcon} />}
+        titleText={allowCreate ? 'Start by creating your project' : 'Start by requesting a project'}
+        icon={
+          <EmptyStateIcon
+            icon={() => (
+              <img style={{ height: 320 }} src={projectsEmptyStateImg} alt="no projects" />
+            )}
+          />
+        }
         headingLevel="h2"
       />
-      <EmptyStateBody>
-        {allowCreate
-          ? `To get started, create a data science project${
-              isJupyterEnabled ? ' or launch a notebook with Jupyter' : ''
-            }.`
-          : `To get started, ask your ${ODH_PRODUCT_NAME} admin for a data science project${
-              isJupyterEnabled ? ' or launch a notebook with Jupyter' : ''
-            }.`}
+      <EmptyStateBody data-testid="projects-empty-body-text">
+        Projects allow you and your team to organize and collaborate on resources within separate
+        namespaces.{!allowCreate ? ' To request a project, contact your administrator.' : ''}
       </EmptyStateBody>
       <EmptyStateFooter>
         {allowCreate ? (
           <>
             <NewProjectButton
               onProjectCreated={(projectName) => navigate(`/projects/${projectName}`)}
             />
-            <EmptyStateActions>
-              <LaunchJupyterButton variant={ButtonVariant.link} />
-            </EmptyStateActions>
           </>
         ) : (
-          <LaunchJupyterButton variant={ButtonVariant.primary} />
+          <Popover
+            minWidth="400px"
+            headerContent="Your administrator might be:"
+            bodyContent={
+              <TextContent data-testid="projects-empty-admin-help-content">
+                <TextList>
+                  <TextListItem>The person who gave you your username</TextListItem>
+                  <TextListItem>
+                    Someone in your IT department or Help desk (at a company or school)
+                  </TextListItem>
+                  <TextListItem>
+                    The person who manages your email service or web site (in a small business or
+                    club)
+                  </TextListItem>
+                </TextList>
+              </TextContent>
+            }
+          >
+            <Button data-testid="projects-empty-admin-help" isInline variant="link">
+              <Icon isInline aria-label="More info">
+                <OutlinedQuestionCircleIcon />
+              </Icon>
+              <span className="pf-v5-u-ml-xs">{`Who's my administrator?`}</span>
+            </Button>
```

Comment:
Use `Button` `icon` attribute instead of nesting and applying class for spacing.